e2e-tests/cypress/integration/appeal-submission-save-and-continue-navigation-appeal-site.feature

```diff
@@ -9,51 +8,48 @@ The navigation does not depend on the status of a sub-section
 Scenario: Navigation - About the appeal site
   Given the "Site location" is presented
   When the "Site location" is submitted with valid values
-  Then the "Site ownership" is presented
+  Then the user is navigated to "Site ownership"
 
 Scenario: Navigation - Site ownership with YES selected
   Given the "Site ownership" is presented
   When the "Site ownership" is submitted with a YES value
-  Then the "Site access" is presented
+  Then the user is navigated to "Site access"
 
 Scenario: Navigation - Site ownership with NO selected
   Given the "Site ownership" is presented
   When the "Site ownership" is submitted with a NO value
-  Then the "Site ownership certb" is presented
+  Then the user is navigated to "Site ownership certb"
 
-#https://appeals-dev.planninginspectorate.gov.uk/appellant-submission/site-ownership-certb
-# 'Site ownership certb' is taken from the URL
 Scenario: Navigation - Site ownership certb with YES selected
-  Given the "Site ownership certb" is presented
+  Given the "Site ownership" is presented
+  When the "Site ownership" is submitted with a NO value
+  Then the user is navigated to "Site ownership certb"
   When the "Site ownership certb" is submitted with a YES value
```

Comment:
I think we should have only one When line in each scenario.

Reply:
OK - not sure how to structure this one. Any clarification appreciated.

This particular feature is to test `/appellant-submission/site-ownership-certb`.

It's possible to browse directly to that page, but if you submit the form without first answering `/appellant-submission/site-ownership` then the API call will fail.

Essentially the first set of `given` / `when` / `then` steps are the mandatory setup for being able to do the actual `when` step that this test really cares about. 

I wasn't sure how to represent this - how would you go about this one? thanks!

Reply:
Everything before the last When event could be considered preparing the state so it could be wrapped up in a single (new) Given step.